schemas/secret.yml

```diff
@@ -0,0 +1,19 @@
+id: http://schemas.taskcluster.net/secrets/v1/secret.json#
+$schema:  http://json-schema.org/draft-04/schema#
+title:        "A TaskCluster Secret"
+description: |
+  Message containing a TaskCluster Secret
+type:         object
+properties:
+  value:
+    description: |
+      The secret value to be encrypted.
+    type:         string
```

Comment:
+1 to this - I expect we'll want to pass, for example, a secret per locale to repack jobs